### DISCUSSION TITLE
Is Gauge Metric Feature Stable?

### DISCUSSION THREAD

**Discussion by Gaganjuneja** [open-telemetry/opentelemetry-java#6272] [Q&A] [answered]
Hi Team,

We are planning to use Gauge type of metrics both synchronous and asynchronous variant in the production. As per the documentation it's still experimental but it's there in the code since 1.10. So just wanted to make sure whether this is still experimental or stable. We are using opentelemetry-java 1.34.0 version.

**Comment by jack-berg** [comment, marked as answer]
The asynchronous gauge instrument is stable. Synchronous gauge is experimental - we're blocked by the synchronous gauge still being [experimental in the spec](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/api.md#gauge), tracking [stability here](https://github.com/open-telemetry/opentelemetry-specification/issues/3893). See this [comment](https://github.com/open-telemetry/opentelemetry-java/pull/5506#issuecomment-1761727698) on how to use the experimental synchronous gauge.
